Package.swift

```diff
@@ -114,13 +114,16 @@ let package = Package(
         .package(url: "https://github.com/slashmo/opentelemetry-swift.git", .upToNextMinor(from: "0.1.1")),
         
         // Apodini Migrator
-        .package(url: "https://github.com/Apodini/ApodiniMigrator.git", .upToNextMinor(from: "0.2.0")),
+        .package(url: "https://github.com/Apodini/ApodiniMigrator.git", .branch("develop")),
```

Comment:
Created the 0.2.2 release: https://github.com/Apodini/ApodiniMigrator/releases/tag/0.2.2

```suggestion
        .package(url: "https://github.com/Apodini/ApodiniMigrator.git", .upToNextMinor(from: "0.2.2")),
```

Reply:
Great! Thank you 🚀